packages/mdl-card/README.md

```diff
@@ -0,0 +1,110 @@
+# mdl-card
+
+MDL card is a component that implements the
+[Material Design card component](https://spec.googleplex.com/quantum/components/cards.html), and makes it available to
+developers as a set of CSS classes.
+
+## Installation
+
+> Note: Installation via the npm registry will be available after alpha.
+
+
+## Usage
+
+```html
+<div class="mdl-card demo-card">
```

Comment:
nit: instead of having `demo-card` as a class (which users might think they have to add) maybe `id="example"` instead? Or, just omit the additional class altogether?